src/core/tests/frontend/onnx/CMakeLists.txt

```diff
@@ -18,12 +18,13 @@ install(TARGETS ${TARGET_NAME}
         COMPONENT tests
         EXCLUDE_FROM_ALL)
 
-set(TEST_ONNX_MODELS_DIRNAME ${TEST_MODEL_ZOO}/src/core/models/onnx)
+set(TEST_ONNX_MODELS_DIRNAME ${TEST_MODEL_ZOO}/core/models/onnx)
```

Comment:
@tomdol Do we have such tests in CI? From my point of view path to models was incorrect...

Reply:
This binary is not executed in CI (yet). There's a problem with this binary described in 80026 and this task is currently in progress.